Manager/MenuManager.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Grossum\MenuBundle\Manager;
+
+use Grossum\MenuBundle\Handler\MenuHandlerInterface;
+
+class MenuManager
+{
+    /**
+     * @var MenuHandlerInterface[]
+     */
+    protected $handlers = [];
+
+    /**
+     * @param MenuHandlerInterface $menuHandler
+     */
+    public function addMenuHandler(MenuHandlerInterface $menuHandler)
+    {
+        $this->handlers[$menuHandler->getIdentifierClass()] = $menuHandler;
+    }
+
+    /**
+     * @param string $className
+     * @return MenuHandlerInterface
+     */
+    public function getMenuHandler($className)
+    {
+        if (!array_key_exists((string)$className, $this->handlers)) {
```

Comment:
@WhiteEagle88 because it's php. When convert to string, I'm sure that there will be a string.